openfecwebapp/config.py

```diff
@@ -28,7 +28,7 @@
 )
 
 features = {
-    'legal': bool(env.get_credential('FEC_FEATURE_LEGAL', '')),
+    'legal_statutes': bool(env.get_credential('FEC_FEATURE_LEGAL_STATUTES', '')),
```

Comment:
👍 i would just add a request we update the readme to point to config.py to see what flags are available? (rather than updating the readme everytime we have a deprecate/add a flag?)